engines/touche/midi.cpp

```diff
@@ -85,7 +85,7 @@ void MidiPlayer::setVolume(int volume) {
 	// transmit the volume change, even if the current _masterVolume
 	// equals the new master volume. This *could* make a difference in
 	// some situations.
-	// So, we should determine whether Touche requires this behavioral
+	// So, we should determine whether Touché requires this behavioral
```

Comment:
This references the engine, not the game title

Reply:
Replaced with "this engine" to avoid the issue.